conductor/server/tasks/delete_device_deployments.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "name": "delete_device_deployments",
+        "retryCount": 5,
+        "timeoutSeconds": 10,
+        "inputKeys": [],
+        "outputKeys": [],
+        "timeoutPolicy": "RETRY",
+        "retryLogic": "EXPONENTIAL_BACKOFF",
+        "retryDelaySeconds": 60,
+        "responseTimeoutSeconds": 3600
```

Comment:
What's the meaning of "responseTimoeutSeconds" parameter?
Why this timeout is set to one hour?